SimplenoteUITests/AllNotesClass.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  AllNotesClass.swift
+//  SimplenoteUITests
+//
+//  Created by Sergiy Fedosov on 03.02.2021.
+//  Copyright © 2021 Automattic. All rights reserved.
+//
+
+import XCTest
+
+class AllNotes {
+
+    class func openNote(noteName: String) {
+        app.tables.cells[noteName].tap()
+    }
+    
+    class func isOpen() -> Bool {
+        return app.navigationBars[uidNavBar_AllNotes].exists
+    }
+
+    class func open() {
+        if !isOpen() {
+            app.navigationBars.element.buttons[uidButton_Menu].tap()
+            app.tables.staticTexts[uidButton_AllNotes].tap()
+        }
```

Comment:
Nitpick: Swift has a `guard` statement that we could use here to make the code (in my opinion) easier to read and (objectively) free from an extra level of indendation:

```suggestion
        guard isOpen() else { return }

        app.navigationBars.element.buttons[uidButton_Menu].tap()
        app.tables.staticTexts[uidButton_AllNotes].tap()
```

Reply:
Thank you! I never used Swift before, and was not aware of `guard` statement. Applied it there and in a few other places.

It looks like `!isOpen()` still should be used in guard statement.